docs/stacks-guide-aws.md

```diff
@@ -110,10 +110,10 @@ To store the credentials as a secret, run:
 
 ```bash
 # retrieve profile's credentials, save it under 'default' profile, and base64 encode it
-AWS_CREDS_BASE64=$(cat ${HOME}/.aws/credentials | awk '/["$aws_profile"]/ {getline; print $0}' | awk 'NR==1{print "[default]"}1' | base64 | tr -d "\n")
+export AWS_CREDS_BASE64=$(cat ${HOME}/.aws/credentials | awk '/["$aws_profile"]/ {getline; print $0}' | awk 'NR==1{print "[default]"}1' | base64 | tr -d "\n")
```

Comment:
was there no other change to this command to extract the creds besides exporting it? that doesn't seem like it would fix the issue you were seeing

Reply:
just figured it out and pushed the change